LoRaEngine/test/LoRaWan.IntegrationTest/IntegrationTestFixtureCI.cs

```diff
@@ -122,26 +122,48 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        public string GetKey16(int deviceId)
+        {
+            var format = string.IsNullOrEmpty(this.Configuration.DeviceKeyFormat) ? "0000000000000000" : this.Configuration.DeviceKeyFormat;
+            if (format.Length < 16)
+            {
+                format = format.PadLeft(16, '0');
+            }
+
+            return deviceId.ToString(format);
+        }
+
+        public string GetKey32(int deviceId)
+        {
+            var format = string.IsNullOrEmpty(this.Configuration.DeviceKeyFormat) ? "00000000000000000000000000000000" : this.Configuration.DeviceKeyFormat;
```

Comment:
Might become a bit more readable and less prone to error if instead of "00000000000000000000000000000000" you would use new String('0', 32)

Reply:
That is true @TechPreacher. Will make this change in a next PR. Thanks!

Reply:
Or maybe simplify the 2 methods to:

public string GetKey(int deviceId, int length)
{
    var format = string.IsNullOrEmpty(this.Configuration.DeviceKeyFormat) ? new string('0', length) : this.Configuration.DeviceKeyFormat;
    if (format.Length < length)
    {
        format = format.PadLeft(length, '0');
    }

    return deviceId.ToString(format);
}